tests/components/iammeter/test_init.py

```diff
@@ -0,0 +1,17 @@
+"""Test the IamMeter component."""
+from homeassistant.components import iammeter
+
+from tests.async_mock import Mock
+
+
+async def test_setup(hass):
+    """Test setup function."""
+    config = {}
+    assert await iammeter.async_setup(hass, config) is True
+
+
+async def test_async_setup_entry(hass):
+    """Test that it will forward setup entry."""
+    hass = Mock()
+    assert await iammeter.async_setup_entry(hass, {}) is True
```

Comment:
Don't interact with integration details.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations